src/Core/ServiceWrapper_dotNET4/packages.config

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="ICSharpCode.SharpZipLib.dll" version="0.85.4.369" targetFramework="net40" />
```

Comment:
Added to ServiceWrapper/packages.config as well, is that what you meant.

Reply:
There is a ILMerge command in the csproj file's build section which links all the dlls into a single executable. This library needs to be referenced there. checkout `winsw_dotNET4.csproj:142`

Reply:
Yes, now there is some duplication in configs now. It may be confusing, but ILMerge is the only way to get such executable with the current project layout and .NET2

Reply:
Ok so this will be picked up automatically by ILMerge as it now added into the packages.config. There is nothing more for me to do, am I correct?